webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DeploymentService.scala

```diff
@@ -1748,12 +1748,12 @@ object RuleExpectedReportBuilder extends Loggable {
         case (m, originalVariables) if (m.isEmpty && originalVariables.nonEmpty)                                          =>
           (Seq(DEFAULT_COMPONENT_KEY), originalVariables.values.flatMap(_.values).toSeq) // this is an autobounding policy
         case (variables, m) if (m.isEmpty)                                                                                =>
-          PolicyGenerationLogger.warn(
+          PolicyGenerationLogger.debug(
             s"Somewhere in the expansion of variables, the bounded variable ${boundingVar} for ${vars.trackerVariable.spec.name} in Directive ${directiveId.serialize} appeared, but was not originally there"
           )
           (variables.values.flatMap(_.values).toSeq, variables.values.flatMap(_.values).toSeq) // this is an autobounding policy
         case (variables, originalVariables)                                                                               =>
-          PolicyGenerationLogger.warn(
+          PolicyGenerationLogger.debug(
             s"Expanded and unexpanded values for bounded variable ${boundingVar} for ${vars.trackerVariable.spec.name} in Directive ${directiveId.serialize} have not the same size : ${variables.values} and ${originalVariables.values}"
```

Comment:
this can now happen often since we correct earlier the real content of tracking key